code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -59,6 +59,9 @@
 	var/mob/living/carbon/C = new_owner.current
 	if(!istype(C))
 		return
+	if(is_banned_from(C.ckey, ROLE_BRAINWASHED))
+		to_chat(admin, "[C] is banned from being brainwashed!")
+		return
```

Comment:
If an admin wants to add an antag datum to someone, they should be allowed to whether that person is banned or not.

It could instead give a confirmation prompt to the admin to allow them to proceed anyway or back out of it - But no other `admin_add()` proc has this check because admins rule all.

Reply:
ok i just won't do it then